examples/spark2/src/main/scala/org/apache/carbondata/examples/S3Example.scala

```diff
@@ -18,52 +18,50 @@ package org.apache.carbondata.examples
 
 import java.io.File
 
-import org.apache.hadoop.fs.s3a.Constants.{ACCESS_KEY, ENDPOINT, SECRET_KEY}
 import org.apache.spark.sql.{Row, SparkSession}
 import org.slf4j.{Logger, LoggerFactory}
 
-import org.apache.carbondata.core.constants.CarbonCommonConstants
+import org.apache.carbondata.spark.util.CarbonSparkUtil
 
 object S3Example {
 
-  /**
-   * This example demonstrate usage of
-   * 1. create carbon table with storage location on object based storage
-   * like AWS S3, Huawei OBS, etc
-   * 2. load data into carbon table, the generated file will be stored on object based storage
-   * query the table.
-   *
-   * @param args require three parameters "Access-key" "Secret-key"
-   *             "table-path on s3" "s3-endpoint" "spark-master"
-   */
+   /**
+    * This example demonstrate usage of
+    * 1. create carbon table with storage location on object based storage
+    * like AWS S3, Huawei OBS, etc
+    * 2. load data into carbon table, the generated file will be stored on object based storage
+    * query the table.
+    *
+    * @param args require three parameters "Access-key" "Secret-key"
+    *             "table-path on s3" "s3-endpoint" "spark-master"
+    */
```

Comment:
@xiaohui0318 please check the indent of comments, it needs to remove one blank.
Check other comments.